templates/rust-shuttle/src/utils.rs

```diff
@@ -0,0 +1,17 @@
+use rand::random;
+
+pub fn send_notification(greeting_id: &str, name: &str) {
+    if random::<f32>() < 0.5 {
+        println!("👻 Failed to send notification: {} - {}", greeting_id, name);
+        panic!("Failed to send notification: {} - {}", greeting_id, name);
```

Comment:
not a good idea, convert this to error. I'm not even sure we catch panics actually.

Reply:
one thing you can do to make this code better overall is the following:

Change the signature of this function to 

```
pub async fn send_notification(greeting_id: &str, name: &str) -> anyhow::Result<()>
```

And then the context run should be as easy as

```
ctx.run(|| send_notification(&greeting_id, &name)).await?
```